determined-ai/determined
# chore: Reimplement experiments store with observable

## Description

[WEB-793]





## Test Plan
- Check if CI are all green
- Check if everything works as expected
  - Go to Cluster page
  - Check if `Active Experiments` card is updated correctly
- Check if there's no performance regression 




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.



[WEB-793]: https://determinedai.atlassian.net/browse/WEB-793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
